### PR TITLE
Homogenize DLMFtex2Wiki.py and tex2Wik

### LINKED ISSUE #41: Homogenize DLMFtex2Wiki.py and tex2Wiki
Make DLMFtex2Wiki.py look like tex2Wiki.
- [x] import tex2Wiki (for now)
- [x] Identify and then remove all subroutines from DLMFtex2Wiki (such as is_number) that are exactly the same in tex2Wiki.
- [x] adjust the output from mmd to xml
- [ ] remove `{\displaystyle` ... `}` wrapper
- [x] rename jsahlfkjsd -> iterations in tex2Wiki
- [ ] make `DLMF` (in `DLMFtex2Wiki`) accept 3 arguments like `readin` (in `tex2Wiki`) i.e., `ofname,glossary,mmd`
- [ ] remove all commented code such as  `#tester=open("testData.txt",'w')`
- [x] replace `wiki.write` with `append_text`
- [ ] convert `bof` `eof` to `append_revision`

You should make a commit after every step.

### PR DESCRIPTION
**Pull request by physikerwelt**
Close #41

### CODE REVIEW

**Comment by physikerwelt** [comment]
@pwang00 Please review the changes I made. We should test the result tomorrow together on the test data. (I currently do not have any data im my data folder, so I can't test it right now)

**Comment by pwang00** [comment]
I just took a look, functionality wise, everything seems good.  However, I think we should delete the

    wiki = ".../../data/ZE.4.xml"

lines, since they don't really serve any purpose after wiki.write was replaced with append_text.

File: Azeem/src/DLMFtex2Wiki.py [hunk 1]
```diff
@@ -1,795 +1,794 @@
...
-    for iterations in range(0,1):
-        try:
-            tex=open(sys.argv[1],'r')
-            wiki = open(sys.argv[2], 'w')
-            glossary = open(sys.argv[3],"r")
-            main=open(sys.argv[4],"r")
-            lLinks = open(sys.argv[5], 'r')
-        except (IOError,IndexError):
-            tex = open("../../data/ZE.3.tex","r")
-            wiki = open("../../data/ZE.4.xml","w")
```
Comment (physikerwelt) [inline Azeem/src/DLMFtex2Wiki.py]: @pwang00 the red text on the left shows the code that was deleted. The green text on the right, shows was added instead.

**Comment by pwang00** [comment]
Everything looks good.  Should we merge?

**Comment by physikerwelt** [comment]
Yes

**Comment by pwang00** [comment]
Merged #48

**Pull request merged**